netspresso/benchmarker/v2/benchmarker.py

```diff
@@ -0,0 +1,199 @@
+from netspresso.enums.device import (
+    DeviceName,
+    HardwareType,
+    SoftwareVersion,
+)
+from netspresso.metadata.benchmarker import BenchmarkerMetadata
+from netspresso.utils import FileHandler, check_credit_balance
+from netspresso.utils.metadata import MetadataHandler
+
+
+class BenchmarkerV2:
+    def __init__(self, token_handler: TokenHandler, user_info: UserResponse) -> None:
+        """Initialize the Benchmarker."""
+
+        self.token_handler = token_handler
+        self.user_info = user_info
+
+    def get_benchmark_task(self, benchmark_task_id: str) -> BenchmarkTask:
+        """Get information about the specified benchmark task using the benchmark task UUID.
+
+        Args:
+            benchmark_task_uuid (str): Benchmark task UUID of the benchmark task.
+
+        Raises:
+            e: If an error occurs while retrieving information about the benchmark task.
+
+        Returns:
+            BenchmarkTask: Model benchmark task object.
+        """
+
+        self.token_handler.validate_token()
+
+        response = launcher_client_v2.benchmarker.read_task(
+            access_token=self.token_handler.tokens.access_token,
+            task_id=benchmark_task_id,
+        )
+        return response.data
+
+    def benchmark_model(
+        self,
+        input_model_path: str,
+        target_device_name: DeviceName,
+        target_software_version: Optional[Union[str, SoftwareVersion]] = None,
+        target_hardware_type: Optional[Union[str, HardwareType]] = None,
+        wait_until_done: bool = True,
+    ) -> BenchmarkTask:
+        """Benchmark the specified model on the specified device.
+
+        Args:
+            input_model_path (str): The file path where the model is located.
+            target_device_name (DeviceName): Target device name.
+            target_software_version (Union[str, SoftwareVersion], optional): Target software version. Required if target_device_name is one of the Jetson devices.
+            target_hardware_type (Union[str, HardwareType], optional): Hardware type. Acceleration options for processing the model inference.
+            wait_until_done (bool): If True, wait for the conversion result before returning the function.
+                                If False, request the conversion and return the function immediately.
+
+        Raises:
+            e: If an error occurs during the benchmarking of the model.
+
+        Returns:
+            Dict: Model benchmark task dictionary.
+        """
+
+        FileHandler.check_input_model_path(input_model_path)
+
+        self.token_handler.validate_token()
+
+        file_name = "benchmark"
+
+        try:
+            folder_path = Path(input_model_path).parent
+
+            benchmarker_metadata = BenchmarkerMetadata()
+            metadatas = []
+
+            if FileHandler.check_exists(folder_path / f"{file_name}.json"):
+                metadatas = MetadataHandler.load_json(folder_path / f"{file_name}.json")
```

Comment:
p2.
Need to change 'folder_path / f"{file_name}.json"' to as a variable.

Reply:
Thanks for your feedback. I’ve updated the code as per your suggestions. 👍